Cython/Build/BuildExecutable.py

```diff
@@ -13,13 +13,20 @@
 
 import sys
 import os
-from distutils import sysconfig
+if sys.version_info < (3, 9):
+    from distutils.sysconfig import get_python_inc
+    from distutils.sysconfig import get_config_var as sysconfig_get_config_var
+else:
+    # sysconfig can be trusted from cpython >= 3.8.7
+    from sysconfig import get_path
+    from sysconfig import get_config_var as sysconfig_get_config_var
+    get_python_inc = lambda: get_path('include')
```

Comment:
I suggest turning this around. Since the old implementation (<3.9) will go away at some point, I'd change the code to use only the new style, and adapt the old imports to match what we now use. That way, we can just remove the old imports at some point, rather than having to change the implementation again in the future.

Reply:
fixed